xds/src/main/java/io/grpc/xds/LookasideLb.java

```diff
@@ -49,31 +52,47 @@
 import io.grpc.xds.LocalityStore.LocalityStoreFactory;
 import io.grpc.xds.XdsClient.EndpointUpdate;
 import io.grpc.xds.XdsClient.EndpointWatcher;
+import io.grpc.xds.XdsClient.RefCountedXdsClientObjectPool;
+import io.grpc.xds.XdsClient.XdsClientFactory;
 import io.grpc.xds.XdsLoadBalancerProvider.XdsConfig;
+import io.grpc.xds.XdsSubchannelPickers.ErrorPicker;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.logging.Logger;
 import javax.annotation.Nullable;
 
 /** Lookaside load balancer that handles balancer name changes. */
-final class LookasideLb extends ForwardingLoadBalancer {
+final class LookasideLb extends LoadBalancer {
 
+  private final ChannelLogger channelLogger;
   private final EdsUpdateCallback edsUpdateCallback;
-  private final GracefulSwitchLoadBalancer lookasideChannelLb;
+  private final GracefulSwitchLoadBalancer switchingLoadBalancer;
   private final LoadBalancerRegistry lbRegistry;
   private final LocalityStoreFactory localityStoreFactory;
   private final LoadReportClientFactory loadReportClientFactory;
+  private final Bootstrapper bootstrapper;
 
-  private String balancerName;
+  /** Most recent XdsConfig. */
```

Comment:
nit: use `//` for one line comment. 

Usually I do not use `/* ... multiple lines ... */` and still use `//` for multi-line comments to explain implementation logic.  I only use `/** ... */` for Javadoc/head comments.

Reply:
I intended to add them as javadoc here. Did you mean it shouldn't be javadoc because it's a private field?

Reply:
Yes.

Reply:
Used `//`.